src/MLJGLMInterface.jl

```diff
@@ -327,4 +327,301 @@ metadata_model(
+  "positive" (second) class.
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `features`: The names of the features used during model fitting
+- `coef`: The linear coefficients determined by the model
+- `intercept`: The intercept determined by the model
+
+# Report
+
+The fields of `report(mach)` are:
+
+- `deviance`: Measure of deviance of fitted model with respect to
+  a perfectly fitted model. For a linear model, this is the weighted
+  residual sum of squares
+- `dof_residual`: degrees of freedom for residuals, when meaningful
+- `stderror`: standard errors of the coefficients
+- `vcov`: estimated variance-covariance matrix of the coefficient estimates
+
+# Examples
+
+```
+using MLJ
+using CategoricalArrays
+using ScientificTypes
+CLF = @load LinearBinaryClassifier pkg=GLM
+clf = CLF(fit_intercept=false)
+
+X, y = @load_iris
+
+# needs binary classification
+idx = findall( x -> string(x) != "setosa", y)
+y = y[idx]
+scitype(y)
+droplevels!(y) # drop unused levels
+scitype(y) # need multiclass 2
+X = map( x -> x[idx], X)
+mach = machine(clf, X, y) |> fit!
+
+Xnew = (sepal_length = [6.4, 7.2, 7.4],
+        sepal_width = [2.8, 3.0, 2.8],
+        petal_length = [5.6, 5.8, 6.1],
+        petal_width = [2.1, 1.6, 1.9],)
+
+yhat = predict(mach, Xnew) # probabilistic predictions
+pdf(yhat, levels(y)) # probability matrix
+p_virginica = pdf.(yhat, "virginica")
+p_positive_class = predict_mean(mach, Xnew) # probability of "positive" class
+p_virginica == p_positive_class
+
+
+fitted_params(mach).features
+fitted_params(mach).coef # x1, x2, intercept
+fitted_params(mach).intercept
+
+report(mach)
+```
+
+See also
+
+[`LinearRegressor`](@ref), [`LinearCountRegressor`](@ref)
+"""
+LinearBinaryClassifier
+
+"""
+$(MMI.doc_header(LinearCountRegressor))
+
+`LinearCountRegressor` is a [generalized linear model](https://en.wikipedia.org/wiki/Generalized_linear_model#Variance_function), specialised
+to the case of a Count target variable whose conditional distribution
+is Poisson, with canonical log link function. Options exist to
+specify an intercept or offset feature.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+    mach = machine(model, X, y)
+
+Where
+
+- `X`: any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitype with `scitype(X)`
+
+- `y`: is the target, which can be any `AbstractVector` whose element
+  scitype is `Count`; check the scitype with `scitype(y)`
+
+# Hyper-parameters
+
+- `fit_intercept=true`: Whether to calculate the intercept for this model.
+   If set to false, no intercept will be calculated (e.g. the data is expected
+   to be centered)
+- `distribution=Distributions.Poisson()`: The distribution which the residuals/
+   errors of the model should fit.
+- `link=GLM.LogLink()`: The function which links the linear prediction function
+   to the probability of a particular outcome or class.
```

Comment:
Again, we should say what the link options are explicitly:

```suggestion
- `link=GLM.LogLink()`: The function which links the linear prediction function
   to the probability of a particular outcome or class. This should be one of the following: 
   `GLM.IdentityLink()`, `GLM.InverseLink()`, `GLM.InverseSquareLink()`,
   `GLM.LogLink()`, `SqrtLink()`.
```